pkg/kubeapiserver/resourcerest/storage.go

```diff
@@ -70,6 +72,7 @@ func (s *RESTStorage) List(ctx context.Context, _ *metainternalversion.ListOptio
 }
 
 func (s *RESTStorage) list(ctx context.Context, options *internal.ListOptions) (runtime.Object, error) {
+	defaultRemainingItemCount := utilfeature.DefaultFeatureGate.Enabled(defaultfeatures.RemainingItemCount)
```

Comment:
This value is required when remaincount is nil, you don't need to judge at the beginning every time.

Reply:
The variable defaultremainingitemcount means whether it is enabled. Using this name will be a little tempting.